frontends/web/src/containers/SubmitModel.js

```diff
@@ -0,0 +1,194 @@
+    inputFile?.click();
+    const fetchData = async () => {
+      await context.api.getTask(props.match.params.taskCode);
+    };
+    fetchData();
+    const task = fetchData();
+    setLoadingFile(false);
+    console.log(task);
+  };
+
+  const handleSubmitModel = (e) => {
+    e.preventDefault();
+
+    if (inputFile.files.length !== 0) {
+      setLoading(false);
+      const user = context.api.getCredentials();
+      const file = inputFile.files[0];
+      const fetchData = async () => {
+        await context.api.uploadModelUser(
+          file,
+          user.username,
+          file.name,
+          file.type,
+          user.id,
+          props.match.params.taskCode
+        );
+      };
+      const result = fetchData();
+      setLoading(true);
+    } else {
+      alert("Please upload a model");
+      setLoadingFile(true);
+    }
+  };
+  return (
+    <>
+      {loading ? (
+        <Container className="mb-5 pb-5">
+          <Col className="m-auto" lg={9}>
+            <div className="mb-5 text-center">
+              <h1 className="my-4 pt-3 text-uppercase">
+                Upload your own model
+              </h1>
+              <h2 className="task-page-header d-block font-weight-normal m-0 text-reset">
+                Imagine you developed a fancy model, living in
+                /path/to/fancy_project, and you want to share your amazing
+                results on the Dynabench model leaderboard. Dynalab makes it
+                easy for you to do just that
+              </h2>
+            </div>
+            <div className="mt-5">
+              <ul>
+                <li>
+                  <strong>Download dynalab 2.0</strong>
+                  <p>
+                    Please download{" "}
+                    <a
+                      download
+                      href="https://models-dynalab.s3.eu-west-3.amazonaws.com/MT/dynalab-base-mt.zip"
+                    >
+                      {" "}
+                      this
+                    </a>{" "}
+                    project, and unzip in your own computer
+                  </p>
+                </li>
+                <li>
+                  <strong>Understand the folders</strong>
+                  <p>
+                    Inside the repository you will find a folder called app.
+                    Inside this folder you will find a folder called domain.
+                    Finally you will find a file called model.py, this is the
+                    only file that must be modified.
+                  </p>
+                </li>
+                <li>
+                  <strong>Single evaluation function</strong>
+                  <p>
+                    In this file you will find a class called "ModelController"
+                    with only two methods. The class contains only one mandatory
+                    method called "single_evaluation". As its name indicates
+                    this method must receive a single input and return a
```

Comment:
Can we say more about what the input is? Basically an "example", right? I'd say "must receive a single example as input" then